test/bdd/steps/api/search-assertion.js

```diff
@@ -0,0 +1,76 @@
+const { When, Then, Given } = require('@cucumber/cucumber');
+const { expect, assert } = require('chai');
+const utilities = require('../../../utilities/utilities');
+let handlerId, searchResult, searchData, parsedKeywords, publishData;
+
+Given(/^I call search request on node (\d+) with ([^"]*) for the keywords:*$/, { timeout: 120000 }, async function (node, resultType, keywords) {
+    resultType = 'assertions';
+    parsedKeywords = utilities.unpackRawTableToArray(keywords);
+
+    const result = await this.state.nodes[node - 1].client
+        .search(resultType, parsedKeywords[0]).catch((error) => {
+            assert.fail(`Error while trying to search assertion. ${error}`);
+        });
+
+    expect(result.status).to.be.eql(202);
```

Comment:
When adding expect please add a fail message, so it's easier to understand why it failed when we look at the logs later eg:
expect(result.status, 'Expected search result status is 202').to.be.eql(202)
or 
expect(result.status).to.be.eql(202, 'Expected search result status is 202')